### PR TITLE
v9.0.0

### PR DESCRIPTION
**Pull request by connorjclark**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,116 @@
...
+We expect this release to ship in the DevTools of [Chrome 98](https://chromiumdash.appspot.com/schedule), and to PageSpeed Insights within 2 weeks.
+
+## New Contributors
+
+Thanks to our new contributors 👽🐷🐰🐯🐻! 
+
+- Sergii Bondarenko @BR0kEN-
+- Prerana Nawar @prerana1821
+
+## Notable Changes
```
Comment (connorjclark) [inline changelog.md:14]: I want to flesh this out.

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,116 @@
...
+* pwa: remove gauge alias ([#13269](https://github.com/GoogleChrome/lighthouse/pull/13269))
+* flow: remove smooth scrolling ([#13317](https://github.com/GoogleChrome/lighthouse/pull/13317))
+* flow: network throttling settings ([#13305](https://github.com/GoogleChrome/lighthouse/pull/13305))
+* flow: use filmstrip thumbnail in navigation ([#13283](https://github.com/GoogleChrome/lighthouse/pull/13283))
+* flow: fix report anchors ([#13233](https://github.com/GoogleChrome/lighthouse/pull/13233))
+* flow: full page screenshot renderer ([#13276](https://github.com/GoogleChrome/lighthouse/pull/13276))
+* flow: category tooltip highest impact ([#13230](https://github.com/GoogleChrome/lighthouse/pull/13230))
+* flow: import lhr strings ([#13215](https://github.com/GoogleChrome/lighthouse/pull/13215))
+* flow: i18n formatter ([#13190](https://github.com/GoogleChrome/lighthouse/pull/13190))
+* flow: mock styles ([#13220](https://github.com/GoogleChrome/lighthouse/pull/13220))
```
Comment (adamraine) [inline changelog.md:59]: These boys can go in the FR section

File: changelog.md [hunk 3]
```diff
@@ -1,3 +1,116 @@
...
+
+* move to minimum Node 14 ([#13243](https://github.com/GoogleChrome/lighthouse/pull/13243))
+* a11y: add relatedNodes to accessibility audits ([#13193](https://github.com/GoogleChrome/lighthouse/pull/13193))
+
+### Removed Audits
+
+* appcache-manifest: remove audit ([#13287](https://github.com/GoogleChrome/lighthouse/pull/13287))
+* external-anchors-use-rel-noopener: remove audit ([#13298](https://github.com/GoogleChrome/lighthouse/pull/13298))
+* remove the redirect pass and redirects-http audit ([#12643](https://github.com/GoogleChrome/lighthouse/pull/12643))
+
```
Comment (adamraine) [inline changelog.md:24]: Breaking changes section, should we include everything on the list https://github.com/GoogleChrome/lighthouse/issues/12614?

Config changes (https://github.com/GoogleChrome/lighthouse/issues/12828) is not in that that issue but we should probably include it.

File: changelog.md [hunk 4]
```diff
@@ -1,3 +1,116 @@
...
+## ⛏️👷 Fraggle Rock
+
+  Support for auditing user flows ([#11313](https://github.com/GoogleChrome/lighthouse/issues/11313))
+
+* fr: handle 0 throughput in timespan ([#13323](https://github.com/GoogleChrome/lighthouse/pull/13323))
+* fr: clone default categories to avoid modification ([#13337](https://github.com/GoogleChrome/lighthouse/pull/13337))
+* fr: add timespan support to h2 ([#12814](https://github.com/GoogleChrome/lighthouse/pull/12814))
+
+## Deps
```
Comment (adamraine) [inline changelog.md:69]: Emojis for all of these like on https://github.com/GoogleChrome/lighthouse/releases/tag/v8.0.0?

File: changelog.md [hunk 5]
```diff
@@ -1,3 +1,123 @@
+<a name="9.0.0"></a>
+# 9.0.0 (2021-11-12)
+[Full Changelog](https://github.com/GoogleChrome/lighthouse/compare/v8.6.0...v9.0.0)
+
+We expect this release to ship in the DevTools of [Chrome 98](https://chromiumdash.appspot.com/schedule), and to PageSpeed Insights within 2 weeks.
+
+See the [What's new in Lighthouse 9.0 blog post](https://web.dev/blog/lighthouse-9-0/). Continue for the changelog.
```
Comment (brendankenny) [inline changelog.md:7]:
```suggestion
See the [What's new in Lighthouse 9.0 blog post](https://developer.chrome.com/blog/lighthouse-9-0/). Continue for the changelog.
```

**Review by brendankenny — COMMENTED** [review]
What are your thoughts on this format vs breaking out a breaking changes section?

**Comment by connorjclark** [comment]
I think the taxonomy of what component the change occurs in is important to preserve, hence I prefer tagging breaking changes with `[BREAKING]`.

**Review by adamraine — APPROVED** [review]

**Comment by brendankenny** [comment]
> I think the taxonomy of what component the change occurs in is important to preserve, hence I prefer tagging breaking changes with `[BREAKING]`.

I hear you and don't think we should be beholden to the format we landed on for the last few major releases, but FWIW here was the thinking behind it:
- half or more of these changes don't matter to anyone (except _maybe_ us). No one cares that the css grid parameters for metrics were changed (since they maintain the earlier order) or which change brought in translated strings. But we've decided it's nice to have a complete record of changes vs just a highlight reel for easy reference (I'm a big fan) and on the off chance one of these is relevant to _someone_
- of the changes that remain, maybe half of readers are looking for interesting new developments and half are looking for what just broke them and how much extra work the Lighthouse team has decided they have to take on today

For the second group we want to make it as easy as possible to scan for breaking changes, and it's nice to give some level of description of the change. Usually just a sentence suffices (e.g. when we're doing a first look at a Chromium bisect range, it's hugely helpful when a description jumps out as relevant vs clicking through into each change and then each associated bug(s)). Those are incidentally helpful for the first group, who can more easily decide if a change is interesting enough to click through and learn more about.

So however they're categorized, ideally we could still improve scannability and giving (brief) context for these.

**Comment by connorjclark** [comment]
I added more context to two of the breaking change commits. The rest were already self explanatory from the commit message. With a `BREAKING` prefix and more careful attention that the commit summary makes sense with less context, I think the current iteration probably suffices?

I am not beholden to any format, so if you have more changes in mind that's fine, but please send a PR/commit/suggested changes to cut down on back/forth for a proposed alternative format.

Anyone know a way to view GitHb's generated changelog feature? What does that look like? Maybe we should just stop doing any editorializing of our changelog and just rely on the new release articles.

**Comment by connorjclark** [comment]
Pending on the release article being published (FYI, its date needs to be updated too), since we link to it.

**Comment by brendankenny** [comment]
> Pending on the release article being published (FYI, its date needs to be updated too), since we link to it.

that's https://github.com/GoogleChrome/developer.chrome.com/pull/1724 for anyone following along :)

**Pull request merged**